src/tests/ftest/util/ior_utils.py

```diff
@@ -4,19 +4,67 @@
 
 SPDX-License-Identifier: BSD-2-Clause-Patent
 """
-
-
 import re
 import uuid
 import time
 from enum import IntEnum
 
-from command_utils_base import FormattedParameter
+from command_utils_base import FormattedParameter, BasicParameter
 from exception_utils import CommandFailure
 from command_utils import ExecutableCommand
 from general_utils import get_subprocess_stdout
 
 
+def run_ior(test, manager, log, hosts, path, slots, group, pool, container, processes, ppn=None,
+            intercept=None, plugin_path=None, dfuse=None, display_space=True, fail_on_warning=False,
+            namespace="/run/ior/*", ior_params=None):
+    # pylint: disable=too-many-arguments
+    """Run IOR on multiple hosts.
+
+    Args:
+        test (Test): avocado Test object
+        manager (JobManager): command to manage the multi-host execution of ior
+        log (str): log file.
+        hosts (list): hostfile list of hosts
+        path (str, optional): hostfile path. Defaults to None.
+        slots (int, optional): hostfile number of slots per host. Defaults to None.
+        group (str): DAOS server group name
+        pool (TestPool): DAOS test pool object
+        container (TestContainer): DAOS test container object.
+        processes (int): number of processes to run
+        ppn (int, optional): number of processes per node to run.  If specified it will override
+            the processes input. Defaults to None.
+        intercept (str, optional): path to interception library. Defaults to None.
+        plugin_path (str, optional): HDF5 vol connector library path. This will enable dfuse
```

Comment:
Eventually, I could see us replacing this with some sort of `ld_preload` parameter to be generic